tests/test_deployments/conftest.py

```diff
@@ -76,3 +76,17 @@ def wait_for_node(
     success = response.status_code == 200
     assert success, "Tendermint node not running"
     yield
+
+
+@pytest.fixture
+def wait_for_occupied_rpc_port(
+    http_: str, loopback: str, rpc_port: int
+) -> Generator[None, None, None]:
+    """Wait for Tendermint to occupy rpc_port."""
+    i, max_retries = 0, 5
```

Comment:
Make max_retries configurable please. Also 5 seems very low for CI...

Reply:
copied from above